packages/generators/init-template/default/webpack.configjs.tpl

```diff
@@ -64,3 +60,25 @@ module.exports = {
         extensions: ['.tsx', '.ts', '.js'],
     },<% } %>
 };
+
+// Add mode specific configurations below
+// Learn more about mode at https://webpack.js.org/configuration/mode/
+
+const production = {
+    mode: 'production',
+}
+
+const development = {
+    mode: 'development',<% if (devServer) { %>
+    devServer: {
+        open: true,
+        host: 'localhost',
+    },<% } %>
+}
+
+module.exports = function (env, argv) {
+    if (argv.mode == 'production') {
+        return merge(base, production);
+    }
+    return merge(base, development);
+}
```

Comment:
We need format it better :smile: 
No `;` in some places, no newline at the end and etc problems, maybe we can generate config, run prettier and format this file as prettier

Reply:
Yup we can, should I do this in this PR itself or open another PR?

Reply:
I think in separate as this will be a big change in scaffolding process itself.

Reply:
Yep, let' do separately, I will review this deeply today

Reply:
I am fixing things manually for this particular PR. Will open separate PR for prettier.